Pearson-Advance/frontend-app-institution-portal
# feat: Instructor detail card

# Description

This PR adds the instructor details card to class detail view.

This PR resolves [PADV-1031](https://agile-jira.pearson.com/browse/PADV-1031)

## Change log
- New component `src/features/Classes/InstructorCard/index.jsx`
- Added the default inscructor image `src/assets/avatar.svg`
- Added the redirect button in `src/features/Classes/Class/ClassPage/index.jsx`.
- Added config to import assets in `webpack.dev.config.js` and `webpack.prod.config.js`


## Screenshots
### Before
![be](https://github.com/Pearson-Advance/frontend-app-institution-portal/assets/52179095/a1abb207-1c39-4b19-b2dc-63580bb00bff)

### After
![af](https://github.com/Pearson-Advance/frontend-app-institution-portal/assets/52179095/4242cbf5-e27d-4cc4-9be8-43cce56693a9)

![af2](https://github.com/Pearson-Advance/frontend-app-institution-portal/assets/52179095/848bf02b-4ec0-4c2a-90b0-3c7e278fa8dd)



### How to test

Clone the Institution Portal MFE
```Bash
     git clone <url>
```
Install the packages 📦.
```Bash
     npm i
```
Start project.
```Bash
     npm start
```

Finally, go to http://localhost:1980/courses/:courseId/:classId and see the result.
